src/Microsoft.ML.Data/DataLoadSave/Text/TextSaver.cs

```diff
@@ -22,25 +22,33 @@ namespace Microsoft.ML.Data.IO
     [BestFriend]
     internal sealed class TextSaver : IDataSaver
     {
+        internal static class DefaultArguments
+        {
+            internal const char Separator = '\t';
+            internal const bool ForceDense = false;
+            internal const bool OutputSchema = true;
+            internal const bool OutputHeader = true;
+        }
+
         // REVIEW: consider saving a command line in a separate file.
         public sealed class Arguments
         {
             [Argument(ArgumentType.AtMostOnce, HelpText = "Separator", ShortName = "sep")]
-            public string Separator = "tab";
+            public string Separator = DefaultArguments.Separator.ToString();
 
```

Comment:
I think that what we had before was correct, I think that `'\t'.ToString()` simply gives a tab as a string. Here it was meant to be an understandable description of separators. We since moved to separator chars in TextLoader and such, in which case we use the character directly to define the separator. Here I believe we have to keep it like it was "tab".

Reply:
Ok.

---
In reply to: [258714604](https://github.com/dotnet/machinelearning/pull/2630#discussion_r258714604) [](ancestors = 258714604)